marlowe-actus/src/Language/Marlowe/ACTUS/Analysis.hs

```diff
@@ -19,22 +19,25 @@ import           Language.Marlowe.ACTUS.Model.STF.StateTransition      (stateTra
 import           Language.Marlowe.ACTUS.Ops                            (ActusNum (..), YearFractionOps (_y))
 import           Prelude                                               hiding (Fractional, Num, (*), (+), (-), (/))
 
+
 genProjectedCashflows :: ContractTerms -> [CashFlow]
-genProjectedCashflows terms =
+genProjectedCashflows = sampleCashflows (\_ -> RiskFactors 1.0 1.0 1.0 0.0)
```

Comment:
You can also write:
```haskell
sampleCashflows $ const $ RiskFactors 1.0 1.0 1.0 0.0
```

Reply:
or even this is what hlint usually suggests
```haskell
sampleCashflows . const . RiskFactors $ 1.0 1.0 1.0 0.0
```

Reply:
To be frank, I find `sampleCashflows (\_ -> RiskFactors 1.0 1.0 1.0 0.0)` more simple, explicit, and readable  than `const` and function composition.

Reply:
I'm happy with all 3

Reply:
@nau I so missed arguing about identity and const. Enjoy my trololo from good old times: https://groups.google.com/g/scala-language/c/IgMLwK3ztjk